### ISSUE TITLE
Are custom scenes supported?

### ISSUE THREAD

**Issue by atv2016** [rospogrigio/localtuya-homeassistant#604]
Are custom scenes i defined in the tuya app 'imported' ? I see a custom scene under the scenes it shows. Is that the scene i created myself in the tuya app ?

**Comment by ThomasCr** [comment]
no, this is not your custom scene, it meens, that this scene is unkown to the app.
